src/classes/UTIL_RecordTypes.cls-meta.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
-    <apiVersion>33.0</apiVersion>
+    <apiVersion>37.0</apiVersion>
```

Comment:
Sounds good, the v37 pull request is ready, but we have to fix some build issues first.